scipy/optimize/_lsq/trf_linear.py

```diff
@@ -140,8 +140,8 @@ def select_step(x, A_h, g_h, c_h, p, p_h, d, lb, ub, theta):
         return ag
 
 
-def trf_linear(A, b, x_lsq, lb, ub, tol, lsq_solver, lsmr_tol, max_iter,
-               verbose):
+def trf_linear(A, b, x_lsq, lb, ub, tol, lsq_solver, lsmr_tol,
+               lsmr_max_iter, max_iter, verbose):
```

Comment:
Similar comment about ordering of new arguments here, although in this case the function may not be public (if so, perhaps less relevant to fret about the ordering).